index.bs

```diff
@@ -1830,6 +1834,107 @@ To <dfn>get the browsing context info</dfn> given |context|,
+and optionally |history handling| (default: "<code>default</code>") and |ignore
+cache| (default: false):
+
+1. Let |navigation id| be the string representation of a
+   [[!RFC4122|UUID]] based on truly random, or pseudo-random numbers.
+
+1. [=Navigate=] |context| with resource |request|, and using |context| as the
+   [=source browsing context=], with navigation id |navigation id|, and history
+   handling |history handling|. If |ignore cache| is true, the navigation must
+   not load resources from the HTTP cache.
+
+   Issue: property specify how the |ignore cache| flag works. This needs to
+   consider whether only the first load of a resource bypasses the cache
+   (i.e. whether this is like initially clearing the cache and proceeding like
+   normal), or whether resources not directly loaded by the HTML parser
+   (e.g. loads initiated by scripts or stylesheets) also bypass the cache.
+
+1.  Let (|event received|, |navigate status|) be [=await=] given
+    «"<code>navigation started</code>", "<code>navigation failed</code>",
+    and "<code>fragment navigated</code>"» and |navigation id|.
+
+1. Assert: |navigate status|'s id is |navigation id|.
+
+1. If |navigate status|'s status is "<code>complete</code>":
+
+  1. Let |body| be a [=map=] matching the
+     <code>BrowsingContextNavigateResult</code> production, with the
+     <code>navigation</code> field set to |navigation id|, and the
+     <code>url</code> field set to the result of the [=URL serializer=] given
+     |navigate status|'s url.
+
+  1. Return [=success=] with data |body|, and then run the following steps [=in
+     parallel=]:
+
+     1. Run the [=WebDriver-BiDi fragment navigated=] steps given |context|
+        and |navigate status|
+
+   Note: this is the case if the navigation only caused the fragment to
+   change. The parallel steps here ensure that we return the command result
+   before emitting the event, so the navigation id is known.
+
+1. If |navigate status|'s status is "<code>canceled</code>" return [=error=]
+   with [=error code=] [=unknown error=].
+
+   TODO: is this the right way to handle errors here?
+
+1. Assert: |navigate status|'s status is "<code>pending</code>" and
+   |navigation id| is not null.
+
+1. If |wait condition| is "<code>none</code>":
+
+  1. Let |body| be a [=map=] matching the
+     <code>BrowsingContextNavigateResult</code> production, with the
+     <code>navigation</code> field set to |navigation id|, and the
+     <code>url</code> field set to the result of the [=URL serializer=] given
+     |navigate status|'s url.
+
+  1. Return [=success=] with data |body|, and then run the following steps [=in
+     parallel=]:
+
+     1. Run the [=WebDriver-BiDi navigation started=] steps given |context|
+        and |navigate status|
+
+1. Run the [=WebDriver-BiDi navigation started=] steps given |context|
+   and |navigate status|
+
+   Note: this event was previously suppressed to ensure that it would come
```

Comment:
Closing the door to my initial reading that this is about a previous state of the spec:

```suggestion
   Note: This event was previously suppressed to ensure that it comes
```